{
    "type": "bundle",
    "id": "bundle--88b51cc0-b86c-4612-afe2-017dbce02fc7",
    "objects": [
        {
            "type": "vulnerability",
            "spec_version": "2.1",
            "id": "vulnerability--46fbbba7-c7e0-4648-a699-f60d95538e33",
            "created_by_ref": "identity--8ce3f695-d5a4-4dc8-9e93-a65af453a31a",
            "created": "2022-06-25T00:51:01.416144Z",
            "modified": "2022-06-25T00:51:01.416144Z",
            "name": "CVE-2022-30885",
            "description": "** Reserved ** The pyesasky for python, as distributed on PyPI, included a code-execution backdoor inserted by a third party. The current version, without this backdoor, is 1.2.0-1.4.2.",
            "external_references": [
                {
                    "source_name": "cve",
                    "external_id": "CVE-2022-30885"
                }
            ],
            "object_marking_refs": [
                "marking-definition--62fd3f9b-15f3-4ebc-802c-91fce9536bcf"
            ]
        }
    ]
}